packages/common/src/api/tan-query/usePurchases.ts

```diff
@@ -35,7 +35,8 @@ export const usePurchases = (
       QUERY_KEYS.purchases,
       args.userId,
```

Comment:
Going to suggest this now before we get too deep into it. But I think we probably _do_ want to pass the args object as a single thing instead of pulling all the individual keys out into an array. It's really easy to mess up the order of the array and you also lose the keys for everything when looking at the query key in devtools.